Testing/functional/tests/cypress/integration/ui/errors/tooManyRequestsAlerts.js

```diff
@@ -0,0 +1,495 @@
+    });
+});
+
+describe("Public Vaccine Card Downloads", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.enableModules([
+            "Immunization",
+            "VaccinationStatus",
+            "VaccinationStatusPdf",
+            "PublicVaccineDownloadPdf",
+        ]);
+        cy.logout();
+        cy.intercept("GET", "**/PublicVaccineStatus", {
+            fixture: "ImmunizationService/publicVaccineStatusLoaded.json",
+        });
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(fullyVaccinatedPhn);
+        cy.get(dobYearSelector).select(fullyVaccinatedDobYear);
+        cy.get(dobMonthSelector).select(fullyVaccinatedDobMonth);
+        cy.get(dobDaySelector).select(fullyVaccinatedDobDay);
+        cy.get(dovYearSelector).select(fullyVaccinatedDovYear);
+        cy.get(dovMonthSelector).select(fullyVaccinatedDovMonth);
+        cy.get(dovDaySelector).select(fullyVaccinatedDovDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.intercept("GET", "**/PublicVaccineStatus/pdf", {
+            statusCode: 429,
+        });
+        cy.get("[data-testid=save-dropdown-btn]")
+            .should("be.visible", "be.enabled")
+            .click();
+        cy.get("[data-testid=save-as-pdf-dropdown-item]")
+            .should("be.visible")
+            .click();
+        cy.get("[data-testid=genericMessageModal]").should("be.visible");
+        cy.get("[data-testid=genericMessageSubmitBtn]").click();
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+        cy.get("[data-testid=loadingSpinner]").should("not.be.visible");
+    });
+});
+
+describe("Landing Page - Too Many Requests", () => {
+    it("Too Many Requests Banner Appears on 429 Response", () => {
+        cy.intercept("GET", "**/Communication/*", { statusCode: 429 });
+        cy.visit("/");
+
+        cy.contains(
+            "[data-testid=communicationBanner]",
+            "higher than usual site traffic"
+        ).should("be.visible");
+    });
+
+    it("Too Many Requests Banner Doesn't Appear on 200 Response", () => {
+        cy.intercept("GET", "**/Communication/*", { statusCode: 200 }).as(
+            "getCommunication"
+        );
+        cy.visit("/");
+
+        // wait for both Communication calls to complete
+        cy.wait("@getCommunication");
+        cy.wait("@getCommunication");
+
+        cy.contains(
+            "[data-testid=communicationBanner]",
+            "higher than usual site traffic"
+        ).should("not.exist");
+    });
+});
+
+describe("Immunization", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/Immunization?*", {
+            statusCode: 429,
+        });
+        cy.enableModules("Immunization");
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak
+        );
+        cy.checkTimelineHasLoaded();
```

Comment:
Timeline has multiple API calls outside of Immunization.  The 'checkTimeLineHasLoaded' call checks if 'loading-in-progress' is no longer showing.  If you're passing in a fixture with status 429, do you need to check if timeline has loaded?  Wouldn't the check on Line 230 be sufficient?